netstandard/ref/System.IO.Compression.cs

```diff
@@ -4,6 +4,47 @@
 
 namespace System.IO.Compression
 {
+    public partial struct BrotliDecoder : System.IDisposable
```

Comment:
I assume there's some reason this and BrotliEncoder are structs? It seems a little odd, but obviously if that's the way they are in Core, they can't change for Standard. (And I'm sure there *is* a good reason, of course.)

Reply:
See the [API design issue](https://github.com/dotnet/corefx/issues/25785):  

> The goal of the streamless implementation is to provide a non-allocating, performant Brotli implementation free from Streams. It contains simple Compress/Decompress operations that return an enum indicating the success of the operation as well as static CompressFully/DecompressFully operations that allow single-pass compression/decompression without the need for a BrotliEncoder/BrotliDecoder instance.

Reply:
And yes, we can't change the design for .NET Standard.